### DISCUSSION TITLE
How can I make subdocuments with mongodb?

### DISCUSSION THREAD

**Discussion by sawa-ko** [mikro-orm/mikro-orm#2160] [Q&A] [answered]
I want to make a collection in mongodb, that has a comments field, that field must have a collection of subdocuments, but I did not find how to do it with mikro-orm.

**Comment by B4nan** [comment, marked as answer]
better ask things like this on slack, and dont expect much help if you dont provide enough details. provide examples to make things clear and provide the code what have you already tried.

**Reply by sawa-ko** [reply]
I'm following the [Embeddeables](https://mikro-orm.io/docs/next/embeddables) guide
